src/Compilers/VisualBasic/Test/CommandLine/CommandLineTests.vb

```diff
@@ -6129,43 +6134,43 @@ End Class
             Dim expected =
     file.Path & "(4) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(0)
-        ~~~   
+        ~~~
 c:\temp\a\1.vb(10) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(1)
-        ~~~   
+        ~~~
 C:\b.vb(20) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(2)
-        ~~~   
+        ~~~
 C:\B.vb(30) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(3)
-        ~~~   
+        ~~~
 " & Path.GetFullPath(Path.Combine(dir.Path, "..\b.vb")) & "(40) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(4)
-        ~~~   
+        ~~~
 " & Path.GetFullPath(Path.Combine(dir.Path, "..\b.vb")) & "(50) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(5)
-        ~~~   
+        ~~~
 C:\X.vb(60) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(6)
-        ~~~   
+        ~~~
 C:\x.vb(70) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(7)
-        ~~~   
+        ~~~
       (90) : error BC30451: 'Foo' is not declared. It may be inaccessible due to its protection level.
         Foo(9)
-        ~~~   
```

Comment:
My code diff is showing a lot of whitespace changes at the end of lines. Not sure if that's CrLf vs Lf line endings or just trimming superfluous spaces off the end of things but if you could avoid it that would be less noisy. Especially because a lot of these tests are annoyingly particular about character positions and hunting down the difference when a test fails can be such a pain.